src/gc/gcdiag.cpp

```diff
@@ -0,0 +1,237 @@
+#include "gcdiag.h"
```

Comment:
Should this file live under src/vm so that it is structured properly for the local GC?